### ISSUE TITLE
Android - Controls gets misplaced on screen when closing fullscreen in landscape mode.

### ISSUE THREAD

**Issue by ubaidvalere** [TheWidlarzGroup/react-native-video#3377]
# Bug
Android video controls are visible outside the area of video when I exit full screen from landscape mode. (Note - I added a code to make app in portrait mode when closing full screen. This is a required functionality).

## Platform
Which player are you experiencing the problem on:
* Android

## Environment info

Library version: 6.0.0-beta.0
Device: Android 12

## Steps To Reproduce
1. Click full screen button
2. rotate device in landscape mode
3. click full screen button again to exit fullscreen.
4. notice the controls get placed outside the video view.

## Expected behaviour
1. Controls should be in place

## Reproducible sample code

```
<Video
            key={data.id}
            ref={videoRef}
            source={{uri: data.url}}
            style={styles.videoStyle}
            onError={console.log}
            repeat
            playInBackground={false}
            playWhenInactive={true}
            muted={false}
            resizeMode="contain"
            progressUpdateInterval={1000}
            onReadyForDisplay={() => setShowLoader(false)}
            ignoreSilentSwitch="ignore"
            volume={1.0}
            // fullscreenOrientation="landscape"
            preventsDisplaySleepDuringVideoPlayback={true}
            onFullscreenPlayerDidDismiss={() => {
              if (Platform.OS === 'android') {
                Orientation.lockToPortrait();
              }
            }}
            onFullscreenPlayerWillPresent={() => {
              Orientation.unlockAllOrientations();
            }}
            controls={true}
          />
```

## Video sample
[Video Sample](https://drive.google.com/file/d/1NVsIO8Z89OXabw4EfaIymoJCFh2hS4DA/view?usp=sharing)

**Comment by iasreact1** [comment]
Can you elaborate your full code with styling attribute ?

**Comment by ubaidvalere** [comment]
This is the style for video component
  videoStyle: {
    aspectRatio: 16 / 9,
    width: width - 30,
    backgroundColor: '#000',
  },

**Comment by freeboub** [comment]
fix will be available on 6.0.0-beta.5
